test/unit/kubernetes-deploy/kubernetes_resource_test.rb

```diff
@@ -97,6 +97,35 @@ def test_fetch_events_returns_empty_hash_when_kubectl_results_empty
     assert_operator(events, :empty?)
   end
 
+  def test_deprecated_annotation_generates_warning
+    customized_resource = DummyResource.new(definition_extras: build_timeout_metadata_deprecated("60S"))
+    customized_resource.validate_definition(kubectl)
+    assert(customized_resource.has_warnings?, "Deprecated annotation did not generate a warning")
+    assert_equal("kubernetes-deploy.shopify.io/timeout-override is deprecated: "\
+      "Use 'krane.shopify.io' annotations instead",
```

Comment:
I'm worried the  current message could be mistaken for depreciating the entire annotation instead of just the prefix what about something like: `kubernetes-deploy.shopify.io` prefix for annotations deprecated please switch to the `krane.shopify.io` prefix?

Reply:
Great shout, will adjust 👍